src/components/CalendarBodyForMonthView.tsx

```diff
@@ -252,8 +254,8 @@ function _CalendarBodyForMonthView<T extends ICalendarEventBase>({
               showAdjacentMonths ? targetDate.date(d) : d > 0 ? targetDate.date(d) : null,
             )
             .map((date, ii) => (
-              <TouchableOpacity
-                onPress={() => date && onPressCell && onPressCell(date.toDate())}
+              <View
+                key={ii}
```

Comment:
@acro5piano 
Oh I fixed it!
What I wanted to do is to prevent "Event Cell" from pressing not the "Date Cell". I want the Date Cell to be pressed and fire event for it.

Furthermore, I added some feedback when pressing in. 

Please check demo video below!

https://github.com/acro5piano/react-native-big-calendar/assets/64240134/f6d6dc14-f678-4c8c-96f5-24db1e8db735